source/includes/_api.md

```diff
@@ -173,7 +175,8 @@ curl https://api.satoshipay.io/v2/goods/ \
        "price": 200000000,
        "asset": "XLM",
        "title": "Nihil placeat sapiente ut eaque assumenda et reprehenderit quos ab.",
-       "url": "http://example.org/post1"
+       "url": "http://example.org/post1",
+       "purchaseValidityPeriod": "1m"
```

Comment:
`"1m"` might not be the best sample, since users might think it means "1 month" (it's 1 minute).

Reply:
sure, but they get to see the response where it says 60000. Should be a valuable lesson for all ;)